### PR TITLE
DMT-47 Update logic for labels responsiveness

### PR DESCRIPTION
**Pull request by krasen86**
## Related issue
- DMT-47

## Proposed changes
- Introduce function hiding the labels' based on the donut-chart size

### CODE REVIEW

**Review by eemilj — COMMENTED** [review]
What if we actually added the functionality of calculateTextVisibility to calculateTextOpacity to return a boolean and just set the text by default?

Kinda like 
.style("opacity", (d) => calculateTextOpacity(d))

calculateTextOpacity(d)
       labelValueLargeEnough = d.data.absPercentage < 5 ? true : false;

because it seems a bit redundant now to have a separate function for it..

File: src/renderer.js [hunk 1]
```diff
@@ -141,14 +141,24 @@ export async function render(donutState) {
                     update
                         .transition("update labels")
                         .duration(animationDuration)
-                        .style("opacity", 1)
-                        .text((d) => calculateTextVisibility(d))
+                        .style("opacity", calculateTextOpacity)
```
[the remark this answers is not in the document]
Comment (eemilj) [reply src/renderer.js:144]: Since we're passing the data now

[the remark this answers is not in the document]
Comment (krasen86) [reply src/renderer.js:144]: You don't really need that, the data is passed

[the remark this answers is not in the document]
Comment (eemilj) [reply src/renderer.js:144]: Ok, just thought that we'd have it consistent with the rest of the code

**Review by eemilj — APPROVED** [review]

**Pull request merged**